src/tools/config_common.h

```diff
@@ -10,28 +53,15 @@ int main(int argc, const char **argv)
 		dbg_msg("usage", "%s FILE1 [ FILE2... ]", argv[0]);
```

Comment:
```suggestion
		dbg_msg("usage", "%s FILE1 [ FILE2... ]", argv[0]);
		dbg_msg("usage", "%s DIRECTORY", argv[0]);
```

Please add this usage to the possible forms, too.